docs/docker.rst

```diff
@@ -8,7 +8,7 @@ Using with Docker
 A docker container exists for easy access to the ``tinygo`` CLI.
 For example, to compile the blinky example from the root of the repository::
```

Comment:
This line is now out of date

Reply:
Good catch!

Reply:
Agreed with the change? Then I will merge it.